### PR TITLE
Refactor `backend`.

### PR DESCRIPTION
**Pull request by Takeshi-Y**
It's so many changes.

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "license": "ISC",
   "dependencies": {
     "bitcoin-core": "^3.0.0",
+    "bitcoinjs-lib": "^5.2.0",
```
Comment (azuchi) [inline backend/package.json:19]:
```suggestion
    "tapyrusjs-lib": "git+https://github.com/chaintope/tapyrusjs-lib.git",
```

File: tapyrus_testnet.conf [hunk 2]
```diff
@@ -1,5 +1,6 @@
 networkid=1939510133
-txindex=1
+txindex=0
+reindex=1
```
[the remark this answers is not in the document]
Comment (Takeshi-Y) [reply tapyrus_testnet.conf:3]: When docker volume using, Tapyrus is not load exists data.
(but I'm not sure if that is the right way...)

[the remark this answers is not in the document]
Comment (azuchi) [reply tapyrus_testnet.conf:3]: However, I think it is not prefer to reindex every time you start up.

[the remark this answers is not in the document]
Comment (Takeshi-Y) [reply tapyrus_testnet.conf]: removed this line.

File: backend/package.json (hunk 1 above)
Comment (Takeshi-Y) [reply backend/package.json]: changed.

**Pull request merged**